canonical/store-components
# Add loading cards

## Done

Added a loading card component.

## QA

- Go to https://store-components-50.demos.haus/?path=/story/loadingcard--default-story
- Check that there is a default loading card component and one with a customisable height
- **Note**: This is lifted straight from the existing charmhub homepage and is not meant to be the final design which can be worked on as we start to use these cards in practice.
